<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddChapter</class>
 <widget class="QWidget" name="AddChapter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Book Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="book_name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Select Chapter number:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="chapter_number"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="chapter_title">
     <property name="placeholderText">
      <string>Chapter Title</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="type_radio">
       <property name="text">
        <string>By Typing</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="file_radio">
       <property name="text">
        <string>By File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="input_method">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="typing_tab">
      <attribute name="title">
       <string>By Typing</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="typing_input">
         <property name="placeholderText">
          <string>Enter Chapter lines...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="file_tab">
      <attribute name="title">
       <string>By File</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Enter File address:&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; (absolute OR relative)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="file_input">
         <property name="placeholderText">
          <string>/projects/Assignment-7/input     OR      input </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="submit">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>typing_input</sender>
   <signal>selectionChanged()</signal>
   <receiver>type_radio</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>file_input</sender>
   <signal>textChanged(QString)</signal>
   <receiver>file_radio</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>type_radio</sender>
   <signal>clicked()</signal>
   <receiver>typing_input</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>file_radio</sender>
   <signal>clicked()</signal>
   <receiver>file_input</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typing_input</sender>
   <signal>textChanged()</signal>
   <receiver>type_radio</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
